src/themes/default/components/core/blocks/Checkout/PersonalDetails.vue

```diff
@@ -238,6 +238,15 @@ export default {
     BaseInput
   },
   mixins: [PersonalDetails],
+  computed: {
```

Comment:
Would be better to have this conditional statement directly in the {{}} using for example ternary operator (condition ? Label1 : Label2)

Reply:
Please remove separate function just for this purpose - the same with the step bumber